eng/Versions.props

```diff
@@ -12,7 +12,7 @@
     <PreReleaseVersionIteration>
     </PreReleaseVersionIteration>
     <ValidateBaseline>true</ValidateBaseline>
-    <IdentityModelVersion>7.0.0</IdentityModelVersion>
```

Comment:
I don't think this will be enough for our templates to pick up my `AadIssuerValidator` fix from https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2361.

The fix is now published as part of [Microsoft.IdentityModel.Validators version 7.0.3](https://www.nuget.org/packages/Microsoft.IdentityModel.Validators/), but none of our package references that use `IdentityModelVersion`, like [Microsoft.IdentityModel.Protocols.OpenIdConnect](https://www.nuget.org/packages/Microsoft.IdentityModel.Protocols.OpenIdConnect) have a dependency on Microsoft.IdentityModel.Validators.

[Microsoft.Identity.Web](https://www.nuget.org/packages/Microsoft.Identity.Web) which is not a dependency of any packages that we ship, but is a dependency of some of our project templates does have a dependency on Microsoft.IdentityModel.Validators, but the latest version of Microsoft.Identity.Web is currently 2.15.1, and it only requires Microsoft.IdentityModel.Validators 6.33.0 which is not recent enough.

@jennyf19 I think we're going to need to update the dependencies for https://www.nuget.org/packages/Microsoft.Identity.Web, publish it to NuGet.org, and then update or dependency versions for those here:

https://github.com/dotnet/aspnetcore/blob/e3aba084cde27f8e718b188e3e9f7239e63d5816/eng/Versions.props#L301-L304

Reply:
Good catch, thanks @halter73. Sounds like we will need to wait for a set of new packages to include in this update. 
@eerhardt FYI, so that you delay your change before this is finalized.

Reply:
ack. We will get started on this.

Reply:
@mkArtakMSFT the release in Id Web (2.15.2) with 7.0.3 IdentityModel is on NuGet
https://github.com/AzureAD/microsoft-identity-web/releases/tag/2.15.2